CHANGES.rst

```diff
@@ -14,6 +14,7 @@ Changelog
 New features:
 
 
+- Add Spanish translation. [macagua] (#1684)
```

Comment:
@macagua Please read the note at the top of this file -- changelog entries should be added in the news folder, not here. This file is only updated at the time that we make a release.

(Your change is obviously not part of the 8.43.0 release, which was already released!)

Reply:
> @macagua Please read the note at the top of this file -- changelog entries should be added in the news folder, not here. This file is only updated at the time that we make a release.
> 
> (Your change is obviously not part of the 8.43.0 release, which was already released!)

@davisagli done with commits [0a94255](https://github.com/plone/plone.restapi/pull/1685/commits/0a94255611b1b11252234471db84cb6534704f37) and [53207ec](https://github.com/plone/plone.restapi/pull/1685/commits/53207ec99e396a21cb5662d0ba07baea8e7e6fb1)